apps/api/src/server/applications/application/application.validators.js

```diff
@@ -226,3 +253,17 @@ export const validateFolderIds = composeMiddleware(
 		.withMessage('Folder must belong to case'),
 	validationErrorHandler
 );
+
+export const validateDocumentGUID = composeMiddleware(
+	param('documentGUID')
+		.custom(validateDocumentGUIDExistsInTable)
+		.withMessage('DocumentGUID must exist in database')
+		.custom(validateDocumentGUIDBelongsToCase)
```

Comment:
I don't think we need to have separate validators here. `validateDocumentGUIDBelongsToCase` will inherently check that the GUID exists in the DB. So I would remove `validateDocumentGUIDExistsInTable` altogether.

Reply:
yes I agree - however, for the tests, the validation error message is the same unless I separate these functions

Reply:
If we are looking to reduce the number of calls we make to the db, then we can get rid of it.
If that doesn't matter, then to keep it would provide more precise feedback?

I am happy to remove it - only because I feel there is a little unnecessary repetition of code.

Reply:
done